src/jmc/compile/command/builtin_function/bool_function.py

```diff
@@ -25,6 +25,19 @@ def call_bool(self) -> tuple[str, bool, list[str]]:
         return f'score {self.args["selector"]} {self.args["objective"]} matches 1..', UNLESS, [
         ]
 
+# please put this somewhere in appropriate place, idk where to put these 2 functions so for now they're here.
+class nbtSource: 
```

Comment:
Class with no instantiation

Reply:
sorry i actually never used classes in python correctly, i had to do some research and now i guess it is what you wanted?